scripts/alert_emails/utils.ts

```diff
@@ -95,6 +103,7 @@ function generateAlertEmailContent(
       `[Alert Feedback] Alert for ${locationName} on ${lastUpdated}`,
     ),
     disclaimer,
+    giveButterDonor,
```

Comment:
Consider passing 2 variables instead, that way the logic stays in one place (the `generateAlertEmailContent` function) instead of passing that to the HTML template.

- `donateUrl`
- `donateText`

Once the experiment is finished, we will just need to delete the logic that creates 2 different `donateUrl` and `donateText` versions and the template will remain unchanged.